frontend/src/model/beta/releases/ReleaseForm.tsx

```diff
@@ -0,0 +1,149 @@
+import { Checkbox, FormControl, FormControlLabel, Stack, TextField, Typography } from '@mui/material'
+import { ChangeEvent, useMemo } from 'react'
+import HelpPopover from 'src/common/HelpPopover'
+import MarkdownDisplay from 'src/common/MarkdownDisplay'
+import ModelImageList from 'src/common/ModelImageList'
+import MultiFileInput from 'src/common/MultiFileInput'
+import RichTextEditor from 'src/common/RichTextEditor'
+import ReadOnlyAnswer from 'src/Form/beta/ReadOnlyAnswer'
+import { FlattenedModelImage } from 'types/interfaces'
+import { ModelInterface } from 'types/v2/types'
+import { isValidSemver } from 'utils/stringUtils'
+
+type ReleaseFormData = {
+  semver: string
+  releaseNotes: string
+  isMinorRelease: boolean
+  artefacts: File[]
+  imageList: FlattenedModelImage[]
+}
+
+type EditableReleaseFormProps =
+  | {
+      editable: true
+      isEdit: boolean
+    }
+  | {
+      editable?: false
+      isEdit?: false
+    }
+
+type ReleaseFormProps = {
+  model: ModelInterface
+  formData: ReleaseFormData
+  onSemverChange: (value: string) => void
+  onReleaseNotesChange: (value: string) => void
+  onMinorReleaseChange: (value: boolean) => void
+  onArtefactsChange: (value: File[]) => void
+  onImageListChange: (value: FlattenedModelImage[]) => void
+} & EditableReleaseFormProps
+
+export default function ReleaseForm({
+  model,
+  formData,
+  onSemverChange,
+  onReleaseNotesChange,
+  onMinorReleaseChange,
+  onArtefactsChange,
+  onImageListChange,
+  editable = false,
+  isEdit = false,
+}: ReleaseFormProps) {
+  const isReadOnly = useMemo(() => editable && !isEdit, [editable, isEdit])
+
+  const handleSemverChange = (event: ChangeEvent<HTMLInputElement>) => {
+    onSemverChange(event.target.value)
+  }
+
+  const handleMinorReleaseChange = (_event: ChangeEvent<HTMLInputElement>, checked: boolean) => {
+    onMinorReleaseChange(checked)
+  }
+
+  const releaseNotesLabel = (
+    <Typography component='label' sx={{ fontWeight: 'bold' }} htmlFor={'new-model-description'}>
```

Comment:
Can we replace the `sx={}` prop with fontWieght prop instead `fontWeight: 'bold'`

Reply:
I've changed this across the app for consistency